json_spec/README.md

```diff
@@ -1,16 +1,2589 @@
-# OCP Hardware Validation Spec Draft 0.1
+# OCP Test and Validation Output Specification
 
-## Specs
-`results_spec.json`: this is the main output schema file, with the objects that should implement by any user of the specs. There are restricted to what is strictly need to identify a validation session, trying to keep them as simple as possible and with more optional parameters.
+---
 
-`extensions`: this directory contains a lot of other objects to include in the validation output to make it more detailed and prone to statistics
+# TODO: revision below? (Siamak)
```

Comment:
Not sure if I follow the reference to Siamak here.  Just remove this TODO?

Reply:
siamak was saying that it should be called "revision" not "version".  i'll just remove the TODO for now, we can resolve later

Reply:
done. please check/resolve

Reply:
Ah, got it.  OK, revision, version... no strong feelings... fine with me.